src/io/fasta.rs

```diff
@@ -857,25 +863,87 @@ impl<W: io::Write> Writer<W> {
     /// assert!(fs::remove_file(path).is_ok());
     /// assert_eq!(actual, expected)
     /// ```
+    pub fn set_linewrap(&mut self, width: usize) {
+        self.linewrap = Some(width);
+    }
```

Comment:
```suggestion
    pub fn set_linewrap(&mut self, linewrap: Option<usize>) {
        self.linewrap = linewrap;
    }
```

Reply:
This is now part of the PR. Let me know if that was done as expected.